sdk/go.mod

```diff
@@ -53,7 +53,7 @@ require (
 	golang.org/x/net v0.34.0
 	golang.org/x/text v0.21.0
 	google.golang.org/grpc v1.69.4
-	google.golang.org/protobuf v1.36.3
+	google.golang.org/protobuf v1.36.5
```

Comment:
Protobuf was updated in some places to 1.36.5, but this seemed to have been missed.